apps/web/pages/[user].tsx

```diff
@@ -20,6 +20,7 @@ import { getUsernameList } from "@calcom/lib/defaultEvents";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { useRouterQuery } from "@calcom/lib/hooks/useRouterQuery";
 import useTheme from "@calcom/lib/hooks/useTheme";
+import logger from "@calcom/lib/logger";
```

Comment:
It seems like we are going to need to wire up something in the logger itself to actually send the exception to Sentry. Were you able to find docs stating that when Sentry is wired up in the Next config, tslog is compatible and will send the exception out?